app/views/decidim/participatory_documents/documents/index.html.erb

```diff
@@ -2,8 +2,17 @@
   <div class="row">
     <%= content_tag(:h3, translated_attribute(document.title), class: "title") if translated_attribute(document.title).present? %>
     <%= content_tag(:p, translated_attribute(document.description).html_safe, class: "description") if translated_attribute(document.description).present? %>
+    <% if session[:preview_mode] && !document.published? %>
```

Comment:
```suggestion
    <% if preview_mode? %>
```
I think it is cleaner to create a helper here and check if the user is an admin or a participatory space admin, I think it could be done like this:
```ruby
def preview_mode?
  return if document.published?

  current_component.manifest.admin_engine && user_role_config.component_is_accessible?(current_component.manifest_name)
end
```